### ISSUE TITLE
Random username generator for Linux VMs should not generate digits as the first character

### ISSUE THREAD

**Issue by SvenAelterman** [microsoft/AzureTRE#3693]
**Describe the bug**
The autogenerated username for Linux VMs might start with a digit. It appears that this is not allowed in Linux because connections to a VM set up like that fail.

**Comment by marrobi** [comment]
Completing this - https://github.com/microsoft/AzureTRE/issues/1148 would resolve this issue.

**Comment by marrobi** [comment]
Closed as username is not from email or upn.

**Comment by marrobi** [comment]
yes, sorry, typo, see https://github.com/microsoft/AzureTRE/pull/4333#issue-2835968007
